framework/field_functions/field_function_grid_based.h

```diff
@@ -76,14 +74,10 @@ class FieldFunctionGridBased : public FieldFunction
    */
   void UpdateFieldVector(const Vec& field_vector);
 
-  /**
-   * Static method to export multiple grid-based field functions.
-   */
-  typedef std::vector<std::shared_ptr<const FieldFunctionGridBased>> FFList;
-
   /**
    * Export multiple field functions to VTK.
    */
+  using FFList = std::vector<std::shared_ptr<const FieldFunctionGridBased>>;
```

Comment:
The doxygen comment is now attached to the using statement, rather then `ExportMultipleToVTK` function.

Reply:
Fixed.